### DISCUSSION TITLE
Compatibility with Oracle DataGuard and how to use

### DISCUSSION THREAD

**Discussion by ratiuseby** [oracle/python-oracledb#177] [Q&A]
Hi, I want to use python-oracledb to connect to Oracle Data Guard (DG) because I saw in the documentation that it is supported (https://python-oracledb.readthedocs.io/en/latest/user_guide/appendix_a.html) but I can't find anything on how to use it. Anyone has some guidelines or small examples on this? I tried searching for anything even slightly related to this but got no results. Thanks in advance

**Comment by anthony-tuininga** [comment]
What do you mean by "how to use it"? Are you referring to how to use DataGuard in general? If so, you're in the wrong forum! If you're simply referring to how to connect to DataGuard then it should work the same way as with any other connection! I am not an expert with this type of database so cannot help you with anything more than that, unfortunately. I would ask in a forum dedicated to Oracle DataGuard. If you find something with another driver (Java, C) then we can help translate that to Python.
